File: DevExpress-Demo/VB/EditorsDemo.Wpf/Controls/DemoModuleControl.vb
Imports DevExpress.Xpf.DemoBase

Namespace EditorsDemo

    Public Class EditorsDemoModule
        Inherits DemoModule

        Protected Overridable ReadOnly Property NeedChangeEditorsTheme As Boolean
            Get
                Return False
            End Get
        End Property
    End Class
End Namespace

Namespace CommonDemo

    Public Class CommonDemoModule
        Inherits EditorsDemo.EditorsDemoModule

    End Class
End Namespace

File: DevExpress-Demo/CS/EditorsDemo.Wpf/Controls/DemoModuleControl.cs
using System;
using System.Windows;
using DevExpress.DemoData.Models;
using DevExpress.Xpf.DemoBase;
using DevExpress.Xpf.Utils.Themes;

namespace EditorsDemo {
	public class EditorsDemoModule : DemoModule {
        protected virtual bool NeedChangeEditorsTheme { get { return false; } }
    }
}
namespace CommonDemo {
    public class CommonDemoModule : EditorsDemo.EditorsDemoModule {
    }
}
